Client/WarFare/UISkillTreeDlg.h

```diff
@@ -18,10 +18,17 @@ const int SKILL_DEF_SPECIAL1 = 2;
 const int SKILL_DEF_SPECIAL2 = 3;
 const int SKILL_DEF_SPECIAL3 = 4;
 
-// MAX_SKILL_KIND_OF	:						 Base Skill 1개, 전문 스킬 4개..			
-// MAX_SKILL_IN_PAGE	:						 한 페이지 내의 아이콘 갯수..				
-// MAX_SKILL_PAGE_NUM	:						 스킬 한개가 가지는 최대 페이지수..		
-
+// Master Scrolls (items consumed)
+constexpr DWORD ITEM_ID_MASTER_SCROLL_WARRIOR		= 379063000;
```

Comment:
Just use uint32_t here for future's sake, rather than unnecessarily keeping it Microsoft-specific. The struct it's being checked against should already be using uint32_t, so it keeps it consistent.

Reply:
Additionally, defining special item IDs here is probably a bad idea - I'd put them in GameDef.h, since they'll most likely need access to other things (e.g. CMagicSkillMng) for its own validation.